mage_ai/frontend/pages/datasets/[slug].tsx

```diff
@@ -233,19 +236,11 @@ function Data() {
     <>
       <FlexContainer justifyContent={'center'}>
         <Flex flex={1}>
-          <SimpleDataTable
-            columnFlexNumbers={[1, 1]}
-            columnHeaders={[{ label: 'Quality Metrics' }]}
-            rowGroupData={metricSample && [metricSample]}
-          />
+          {metricsEl}
```

Comment:
Putting the `SimpleDataTable` here directly instead of `metricsEl` was actually intentional. There does come a point where it's not necessary to declare every single React component inside of a variable. I found myself wanting to know what the code was and having to find the variable elsewhere in the file. Made going thru the code a bit disjointed imo, but might jsut be a preference thing.